requirements-build.txt

```diff
@@ -7,6 +7,7 @@ pytest-timer==1.0.0
 timeout-decorator==0.5.0
 coverage==7.6.1
 pytest-cov==5.0.0
+parameterized
```

Comment:
Add the parameterized package which is the heavily leveraged fulcrum of making dynamic hocon tests easier to put together.  

This is not a fast-moving package so I am not worried about the lack of version pinning here.